src/Utilities/TrackedDependencies/CanonicalTrackedOutputFiles.cs

```diff
@@ -32,12 +32,7 @@ public class CanonicalTrackedOutputFiles
         #region Properties
 
         // Provide external access to the dependencyTable
-#if WHIDBEY_VISIBILITY
-        internal
-#else
-        public
-#endif
-        Dictionary<string, Dictionary<string, DateTime>> DependencyTable
+        internal Dictionary<string, Dictionary<string, DateTime>> DependencyTable
```

Comment:
It looks like you assumed WHIDBEY_VISIBILITY is always true, but I don't see how that was getting set.  Am I missing it?